docs/google-kubernetes-tutorial.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Deploy TiDB to Kubernetes on Google Cloud
+summary: Tutorial for deploying TiDB on Google Cloud using Kubernetes.
+category: operations
+---
+
+# Deploy TiDB, a distributed MySQL compatible database, to Kubernetes on Google Cloud
+
+## Introduction
+
+This tutorial is designed to be [run in Google Cloud Shell](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/pingcap/tidb-operator).  It takes you through these steps:
+
+- Launching a new 3 node Kubernetes cluster (optional)
+- Installing the Helm package manager for Kubernetes
+- Deploying the TiDB Operator
+- Deploying your first TiDB cluster
+- Connecting to TiDB
+- Shutting down down the Kubernetes cluster (optional)
+
+## Select a Project
+
+Please select a project before proceeding.  The approximate cost for running the compute resources in this tutorial is 10 cents/hour.
+
+<walkthrough-project-billing-setup key="project-id">
+</walkthrough-project-billing-setup>
+
+## Enable API Acess
+
+This tutorial will require use of the Compute and Container APIs.  Please enable them before proceeding:
+
+<walkthrough-enable-apis apis="container.googleapis.com,compute.googleapis.com">
+</walkthrough-enable-apis>
+
+## Configure gcloud Defaults
+
+This step defaults gcloud to your prefered project and [zone](https://cloud.google.com/compute/docs/regions-zones/), which will simplify the commands used for the rest of this tutorial:
+
```

Comment:
For commands, please use ``` to highlight the code snippet.

Reply:
This tutorial is designed to be played in the Google Cloud Shell.  Code in ``` works differently to tab indented :(

The tab indented can be clicked on to auto-paste in the shell.  The ``` requires a copy+paste.  I'm not sure why Google did it that way, but this is intentional on our part.

Reply:
Got it. Thanks for the clarification.